src/openrct2/ride/Ride.cpp

```diff
@@ -2078,8 +2078,12 @@ void Ride::Update()
 
     // Various things include news messages
     if (lifecycle_flags & (RIDE_LIFECYCLE_BREAKDOWN_PENDING | RIDE_LIFECYCLE_BROKEN_DOWN | RIDE_LIFECYCLE_DUE_INSPECTION))
-        if (((gCurrentTicks >> 1) & 255) == static_cast<uint32_t>(id))
+    {
+        // Breakdown updates are distributed, only one ride can update the breakdown status per tick.
```

Comment:
But why. That seems like a bad idea. Perhaps if we did `(gCurrentTicks & 255) == (rideId & 255)` then it does not change anything balance wise from the base game but allows for the new limits